app/Repositories/Eloquents/PostEloquentRepository.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/4/18
+ * Time: 6:21 PM
+ */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostEloquentRepository.php</br>**Line**: 9</br>There must be one blank line after the namespace declaration